docs/api/navigators/StackNavigator.md

```diff
@@ -137,7 +137,7 @@ Color for material ripple (Android >= 5.0 only)
 
 #### `gesturesEnabled`
 
-Whether you can use gestures to dismiss this screen. Defaults to true on iOS, false on Android
+Whether you can use gestures to dismiss this screen. Defaults to true on iOS, false on Android. This value when set is only respected in `mode` == `card`.
```

Comment:
@nihgwu, This should already work, to set it in the StackNavigatorConfig as you mentioned:

```
const App = StackNavigator({
  ...
}, {
  navigationOptions: {
    gesturesEnabled: false,
  }
});
```

The default value for every screen navigationOptions comes from the navigator.